scripts/deploy-federated-nginx.sh

```diff
@@ -52,9 +52,9 @@ kubefedctl federate ns ${TEST_NS} --contents --skip-api-resources 'pods,secrets,
 echo
 echo "Checking status of federated resources."
 util::wait-for-condition 'federated deployment status updated in cluster1' \
-    "(kubectl get federateddeployment nginx -n ${TEST_NS} -o jsonpath='{.status.clusters[*].name}' | grep '^cluster1') &> /dev/null" 120
+    "(kubectl get federateddeployment nginx -n ${TEST_NS} -o jsonpath='{.status.clusters[*].name}' | grep '\<cluster1\>') &> /dev/null" 120
```

Comment:
So this fixes the problem? I'm not sure why angle brackets would be included in the output.

Reply:
Until now, I works. I tried several times. It is fine.
`\<word\>` means a word match.